libs/dataset-explorer/src/lib/DataExplorerParentTab.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Pivot, PivotItem, Stack } from "@fluentui/react";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { DataBalanceTab } from "./DataBalanceTab";
+import { dataExplorerParentTabStyles } from "./DataExplorerParentTab.styles";
+import { DatasetExplorerTab } from "./DatasetExplorerTab";
+
+interface IDataExplorerParentTabProps {
+  showDataBalanceExperience: boolean;
+}
+
+interface IDataExplorerParentTabState {
+  activeViewOption: DataExplorerParentTabOptions;
+}
+
+enum DataExplorerParentTabOptions {
+  DatasetExplorer = "DatasetExplorer",
+  DataBalance = "DataBalance"
+}
+
+export class DataExplorerParentTab extends React.Component<
+  IDataExplorerParentTabProps,
+  IDataExplorerParentTabState
+> {
+  public constructor(props: IDataExplorerParentTabProps) {
+    super(props);
+    this.state = {
+      activeViewOption: DataExplorerParentTabOptions.DatasetExplorer
+    };
+  }
+
+  public render(): React.ReactNode {
+    const styles = dataExplorerParentTabStyles();
+
+    if (!this.props.showDataBalanceExperience) {
+      return <DatasetExplorerTab />;
+    }
+
+    return (
+      <Stack>
+        <Stack.Item>
+          <Stack
+            horizontal
+            tokens={{ childrenGap: "s1", padding: "l1" }}
+            className={styles.container}
+          >
+            <Pivot onLinkClick={this.onViewTypeChange}>
+              <PivotItem
+                itemKey={DataExplorerParentTabOptions.DatasetExplorer}
+                headerText={
+                  localization.ModelAssessment.ComponentNames.DataExplorer
+                }
+              />
+              <PivotItem
+                itemKey={DataExplorerParentTabOptions.DataBalance}
+                headerText={
+                  localization.ModelAssessment.ComponentNames.DataBalance
+                }
+              />
+            </Pivot>
+          </Stack>
+        </Stack.Item>
+        <Stack.Item>
+          {this.state.activeViewOption ===
+            DataExplorerParentTabOptions.DatasetExplorer && (
+            <DatasetExplorerTab />
+          )}
+          {this.state.activeViewOption ===
+            DataExplorerParentTabOptions.DataBalance && <DataBalanceTab />}
+        </Stack.Item>
```

Comment:
Any particular reason for rendering these contents separately? You can do this:

```
<PivotItem ...>
    <DatasetExplorerTab />
</PivotItem>
<PivotItem ...>
    <DatasetBalanceTab />
</PivotItem>
```

Reply:
I didn't even realize you can do this! Fixed 😊

- Removed `IDataExplorerParentTabState` completely
- Removed use of StackItems in favor of PivotItems
- Removed `onViewTypeChange` since we are no longer keeping state
- Added a `pivotLabelWrapper` style